quickwin/KAD-4558_adv_inline_image_format

gilbert-hernandez opened the pull request:
[https://stellarwp.atlassian.net/browse/KAD-4558](https://stellarwp.atlassian.net/browse/KAD-4558)

Adds an inline image format for the advanced heading.

mark-c-woodard reviewed (changes requested):
https://www.loom.com/share/ed4c5cb2f9d54a08b9bbcf5664936506

gilbert-hernandez commented:
@mark-c-woodard 
I did get worked out. originally, I updated the component to more closely resemble the core inline image in the way it inserts the image. I also added responsive styles for the width.
